vignettes/corridor-delineation.Rmd

```diff
@@ -14,16 +14,17 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup}
+```{r setup, warning=FALSE, message=FALSE}
 library(CRiSp)
-library(sf)
 library(dplyr)
+library(sf)
+library(osmdata)
 ```
 
-In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS for the analysis. Also, we make sure that we include a given area around the city boundaries.
+In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS for the analysis. A project CRS is required for calculating distances along the street network during delineation. Also, we make sure that we include a given area around the city boundaries.
 
 ```{r variables}
-city_name <- "Bucharest, Romania"   # Be specific and spell as it appears in OSM
+city_name <- "Bucharest, Romania"   # Be specific and spell as in OSM
```

Comment:
```suggestion
city_name <- "Bucharest, Romania"   # Be specific and spell as in OpenStreetMap (OSM)
```
Given the instruction below it would make sense to be consistent in wording used